### ISSUE TITLE
textfield / textarea do not bind variables from view to controller

### ISSUE THREAD

**Issue by cherta** [truenorth/ember-material-lite#22]
First of all thanks for putting this work together. 

I have an `mdl-textfield` that binds his value to a controller variable but when I change the value on the `mdl-textfield` the property in the controller never gets updated. 

It does work the other way around, when I change the property value at the controller level I see the change in the textfield.

Tried also with a regular `{{input type="text"}}` to check if it was my set up but it worked correctly.

```
{{mdl-textfield value=identification}}
{{input type="text" value=identification}}
```

Changing **identification** on the regular input field does the job but not when I try using the `mdl-textfield` component.

**Comment by truenorth** [comment]
Quite possible there are bugs here. I'll take a look at this use case

**Comment by truenorth** [comment]
Confirmed bug

**Comment by truenorth** [comment]
Just released [v0.0.12](https://github.com/truenorth/ember-material-lite/blob/master/CHANGELOG.md#v0012-2015-07-23) with this fix.

**Comment by cherta** [comment]
Thanks @truenorth
